test/serverside.js

```diff
@@ -607,13 +607,84 @@ describe('Managing users', function() {
 	});
 
 	it('ban user', async function() {
+		const client1 = await getTestClientForUser(evilUser);
+		let s;
+		const eventPromise = new Promise(resolve => {
+			s = client1.on('user.banned', e => {
+				expect(e.reason).to.equal('reason');
```

Comment:
needs more event data tests (ie. no channel fields)

Reply:
Sorry missed this comment 🤦‍♂ 

there are no channel fields.

Reply:
probably we need to add channel fields? @ravlio

Reply:
There are two kinds of `ban`, first is ban for channel, second is ban global for app. If user get's banned from channel there should be channel fields. In test you have to make sure there are no channel field for global ban, and there are channel fields for channel ban, I think. Fix me if I'm wrong